Scripts/Modules/TooltipModule/Tooltip.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace UnityEditor.Experimental.EditorVR.UI
 {
-    sealed class Tooltip : MonoBehaviour, ITooltip, ITooltipPlacement
+    public sealed class Tooltip : MonoBehaviour, ITooltip, ITooltipPlacement
```

Comment:
hmmm. Do we want all this to be public? If I go back to Tooltip being internal, I need to also make Affordance, ProxyAnimator, and ProxyUI internal.  So far, we've been treating all public classes as API or something which will be used by third-party packages.

I think all of this stuff should be internal.

Reply:
@AndrewTHEManeri  thoughts?  I'm leaning toward internal as well.

Reply:
Internal